Readme.md

```diff
@@ -42,39 +44,72 @@ Konfiguration im Admin-Bereich
-* Name des Trainers
-* Motto des Trainers
-* Ort, an dem der Trainer seine Angebote bereitstellt - ein Trainer kann das prinzipiell an mehreren Orten tun
-* Allgemeine Informationen über den Trainer und seine Angebote
+### Logout
+Sobald hier geklickt wird, wird der Benutzer sofort ausgeloggt. Die allauth-Settings wurden angepasst, damit keine extra Bestätigungsseite angezeigt wird.
 
+## Trainerliste
+Über die Navigation im Header kommt man zur Trainerliste (`/trainerlist/`). Diese ist für alle Besucher der Seite sichtbar, egal ob eingeloggt oder nicht. Jedoch werden hier nur Trainer gelistet, die ein öffentliches Profil haben.
+Über die Filter in der Liste kann man die Anzeige einschränken. Der Filter "Trainer name" sucht den angegebenen Namen (oder Teil des Namens) in den Vor- und Nachnamen aller Trainer. "Sport" schränkt die Suche auf die angegebene Sportart ein, "Location" sucht in den Orten, in denen der Trainer seine Kurse anbietet.
+Die Anzahl der Suchresultate wird über der Liste angezeigt.
+Über einen Button kommt der Besucher auf das jeweilige Trainerprofil.
 
-### Trainer
-* Trainer können sich über den Link im Header registrieren (`/user/signup/`). Sobald sich ein Benutzer über die Website registriert hat, wird er in der Datenbank mit der Rolle "trainer" angelegt. Nicht über die Website registrierte Benutzer (wie z.B der superuser) werden mit der Rolle "user" angelegt.
-* Registrierte Trainer können sich über den Link im Header einloggen (`/user/login/`). 
-* Eingeloggte Benutzer sehen ihr Avatar im Header und haben Zugriff auf eine Benutzernavigation.
-  * Der Link "Your settings" führt zu den Profil Settings `/user/` und der Benutzer kann hier seine Daten und sein Passwort ändern. Außerdem kann hier ein Avatarbild hochgeladen werden, das im Header angezeigt wird. Bei jedem neuen Avatarupload wird der alte Avatar automatisch aus dem Filesystem gelöscht.
-  * Der Link Logout loggt den aktuellen Benutzer aus.
-  * Benutzer mit der Rolle "trainer" sehen zusätzlich den Link "Your trainer profile" (`/trainer/`), der zum Trainerprofil des eingeloggten Trainers führt. Hier kann der Trainer sein öffentliches Profil anlegen und editieren.
-* Trainer können ihr Profil sichtbar oder unsichtbar schalten. Hierzu können sie eine Checkbox in ihrem Trainerprofil setzen. Unsichtbare Trainer erscheinen nicht in der Trainerliste und andere Benutzer haben auch keinen Zugriff auf das Trainerprofil.
+## Trainerprofile
+Das Trainerprofil zeigt alle Daten für den jeweiligen Trainer. Hier wird das Avatarbild des Trainers und das Motto angezeigt, darunter sieht man, wieviele Benutzer den Trainer favorisiert haben.
 
+Für eingeloggte normale Benutzer (nicht für Trainer oder anonyme Benutzer) wird darunter noch ein Button angezeigt, mit dem ein Trainer favorisiert werden kann. Sobald ein Trainer favorisiert wurde, ändert sich der Button und der eingeloggte normale Benutzer kann den Trainer wieder entfavorisieren. 
```

Comment:
Ist das doppelt zum Inhalt ab Zeile 81?

Reply:
Nein, das ist der Button in der Trainer Detail View. Zeile 81 ist die Favoritenliste für den User.

Reply:
Alles klar.